.github/workflows/ci.yml

```diff
@@ -92,7 +92,7 @@ jobs:
     strategy:
       max-parallel: 4
       matrix:
-        python-version: ['3.8', '3.9', '3.10', '3.11.0-alpha - 3.11.0']
+        python-version: ['3.9', '3.10', '3.11.0-alpha - 3.11.0']
```

Comment:
Should this now change to "3.11.0-rc"?

Reply:
It could, but it doesn't have to. The build step will pull in the most recent tag that matches; if you check the build logs, the most recent build was against 3.11.0rc2. Once 3.11 is finalised, we should be able to add 3.11 without any adornment, and add "3.12.0-alpha - 3.12.0"

Reply:
Oh, I see, that hyphen is actually [a version range](https://github.com/actions/setup-python/blob/main/docs/advanced-usage.md#using-the-python-version-input).